docs/user_guide.md

```diff
@@ -1,11 +1,11 @@
-# Documentation of the general interpolation library 2 - ip2lib
+# Documentation of the general interpolation library - iplib
 
 ## Introduction
 
-The NCEP general interpolation library 2 (ip2lib) contains Fortran 90 subprograms
+The NCEP general interpolation library (ip2lib) contains Fortran 90 subprograms
 to be used for interpolating between nearly all grids used at NCEP.
 The library is particularly efficient when interpolating many fields at one time.
-The library has been extensively tested with AIX and Intel Fortran compilers.
+The library has been extensively tested with GNU and Intel Fortran compilers.
```

Comment:
Well I would strike this line. I'm not sure the library has been "extensively tested" anywhere...